pkg/metrics/metrics.go

```diff
@@ -450,6 +452,15 @@ func sendUsageStats() {
 		metrics["stats.alert_notifiers."+stats.Type+".count"] = stats.Count
 	}
 
+	auth["anonymous_enabled"] = setting.AnonymousEnabled
```

Comment:
I'm not sure how ES store bool values. But I think it would be better to store these values as numbers since it makes it easier to aggregate it with `Sum`.